test/integration/targets/exists/tasks/main.yml

```diff
@@ -0,0 +1,111 @@
+# test code for the exists module
+# (c) 2014, James Tanner <tanner.jc@gmail.com>
```

Comment:
Is Date / Author correct here? I assume this is some % copy pasta, and some % new code.